nifi-frontend/src/main/frontend/apps/nifi-registry/src/app/service/buckets.service.ts

```diff
@@ -37,6 +39,61 @@ export class BucketsService {
         // });
         // return throwError(() => mockError);
 
-        return this.httpClient.get(`${BucketsService.API}/buckets`);
+        return this.httpClient.get<Bucket[]>(`${BucketsService.API}/buckets`);
+    }
+
+    createBucket(request: CreateBucketRequest): Observable<Bucket> {
+        // const mockError: HttpErrorResponse = new HttpErrorResponse({
```

Comment:
Do we want to leave this commented code here? Unless I'm developing/testing a specific error code and the resulting behavior, I usually just stop the backend and attempt to make the request from the UI. Although the status is different (500 internal server error), I'm not seeing the advantage to leaving these comments for future testing.

Reply:
I think we do want to keep these here. They are here to make it easy to test all the error handling scenarios. We don't want to leave the error scenario turned on for the dev server as we want the application to behave normally but we want to be able to quickly test error scenarios.